pages/common/gh-run.md

```diff
@@ -0,0 +1,36 @@
+# gh run
+
+> View, run and watch recent workflow runs from GitHub Actions.
```

Comment:
```suggestion
> View, run and watch recent GitHub Actions workflow runs.
```

Reply:
Looks like the man page called this stuff `run`s, that's why this subcommand is called `run`.